contributors/design-proposals/testing/trigger-pr-tests.md

```diff
@@ -0,0 +1,92 @@
+# Triggering PR tests
+
+## Objective
+
+This document describes a major change to the way the `trigger` plugin determines if test jobs should be started on a pull request (PR).
+
+## Current situation
+
+PR test jobs are started by the trigger plugin on **trusted PR** events, or when a untrusted PR becomes trusted.
+> A PR is considered trusted if the author is a member of the *trusted organization* for the repository or if such a member has left an `/ok-to-test` command on the PR.
+
+You can easily spot an untrusted PR by its `needs-ok-to-test` label.
```

Comment:
We should continue to provide people with an expedient means of searching for PRs that need the `ok-to-test` label.

There is not a way to search for PRs that are "authored by non-members without an `ok-to-test`" label.

Reply:
Indeed, so we might propose `ok-to-test` required for everybody.